content/sensu-core/0.29/guides/troubleshooting.md

```diff
@@ -97,6 +97,65 @@ Sensu staff, or community members may ask to see your logs. You can view them at
+
+Consider the following scenario: Sensu has been installed, has been verified to be working correctly (alerts are seen in the dashboard), and is configured to send alerts via the [mailer handler][14]. However, mail doesn't appear to be coming through.
+
+We'll start by crafting a test command to send to the local socket:
+
+echo ‘{“name”: “testing”, “output”: “THIS IS AN ERROR”, “status”: 2, “refresh”: 10, “handlers”: [“mailer”]}’ | nc localhost 3030
+
+_WARNING: Successfully submitting a check result this way will be indicated by `ok` being printed on the next line -- typically this is appears ahead of the command prompt so it can be easily missed. See below for an example._
+
+{{< highlight shell >}}
+[root@sensu ~]# echo '{"name": "testing_error", "status": 2, "output": "An error event should be created", "refresh": 10, "handlers": [ "mailer"]}' | nc 127.0.0.1 3030
+ok{{< /highlight >}}
+
+
+_NOTE: A successfully submitted check result will also be logged to the sensu-client.log (viewable while in debug logging mode)._
+
+{{< highlight JSON >}}
+{"timestamp":"2018-10-12T18:28:43.204565+0000","level":"info","message":"publishing check result","payload":{"client":"sensu-enterprise","check":{"name":"testing_error","output":"its just a test","status":2,"handler":"opsgenie","opsgenie":{"tags":{"this":"is wrong"}},"refresh":2,"executed":1539368923,"issued":1539368923}}}{{< /highlight >}}
+
+Review the logs on the Sensu server to determine if the issue is making it through to the server.
+Grep for the error message, specifically the check "name" attribute.
+
+{{< highlight json >}}
+{"timestamp":"2018-10-11T11:02:00.576261-0500","level":"info","message":"processing event","event":{"id":"f4a9453f-ac70-4e91-a601-a97ff31c589a","client":{"name":"sensu.test.local","address":"192.168.156.176","environment":"testing","subscriptions":["dev","linux-hosts","roundrobin:web_probe","client:sensu.test.local"],"version":"1.5.0","timestamp":1539273717},"check":{"name":"testing","output":"THIS IS AN ERROR","status":2,"refresh":10,"executed":1539273720,"issued":1539273720,"type":"standard","history":["2"],"total_state_change":0},"occurrences":1,"occurrences_watermark":1,"last_ok":null,"action":"create","timestamp":1539273720,"last_state_change":1539273720,"silenced":false,"silenced_by":[]}}{{< /highlight >}}
+
+It's also recommended that you note the event ID, as this persists and allows you to track an event throughout its lifecycle.
+
+{{< highlight shell >}}
+"event":{"id":"f4a9453f-ac70-4e91-a601-a97ff31c589a"}{{< /highlight >}}
+
+Ensure that the event is being handled by the mailer handler (you can do this by searching for the `event_id` and looking at additional log entries to confirm that the event is handled as expected). It's worth noting that in the mailer example, the issue typically seen is a lack of notifications due to the handler performing its own event filtering. The solution is to set a low refresh value in the check configuration. 
```

Comment:
Is there an issue open for this community mailer handler behavior? How will we know when we can remove this aside from the guide?